### ISSUE TITLE
make install does not seem to copy libgap_alsa.so to  /usr/lib/gogglesmm

### ISSUE THREAD

**Issue by sophom** [gogglesmm/gogglesmm#60]
I finally got the development branch compiling on Ubuntu 16.04 but I can't listen to music.

I get this error when I try to play:

"Failed to load output plugin: /usr/lib/gogglesmm/libgap_alsa.so 
Reason: /usr/lib/gogglesmm/libgap_alsa.so: cannot open shared object file: No such file or directory"

libgap_alsa.so	libgap_pulse.so  libgap_wav.so all get made in the src directory. When I do a sudo make install though they don't seem to get copied over in to /usr

If I create the gogglesmm directory in /usr/lib and copy the lib* files in there, I get this error message:

"Failed to load output plugin: /usr/lib/gogglesmm/libgap_alsa.so.
Reason: /usr/lib/gogglesmm/libgap_alsa.so: undefined symbol: _ZN2ap120OutputThread21notify_disable_volumeEv"

**Comment by gogglesguy** [comment]
@sophom I think you may be using a old version of the development branch. You should pull the latest code.

I'm now also maintaining builds for Ubuntu btw:
https://launchpad.net/~s.jansen/+archive/ubuntu/gogglesmm

**Comment by sophom** [comment]
Nice! I just installed it from the PPA. THANK YOU!

At first I got "FOX Header (v1.7.55) / Library (v1.7.56) mismatch!"... This when running the executable from the PPA. I was the one that installed FOX 1.7.56 from source while I was trying to build the development release.. Anyway the executable version from the development branch started working at that point but since I didn't want an update to overwrite it by accident I ended up removing FOX 1.7.56 and reinstalling from the PPA. Everything works great now.
